test/e2e/common.go

```diff
@@ -17,9 +17,38 @@ limitations under the License.
 package e2e
 
 import (
+	"context"
 	"fmt"
 
 	. "github.com/onsi/ginkgo" // nolint:gci
+	corev1 "k8s.io/api/core/v1"
```

Comment:
In general a comment about this file, any reason why we are moving the test vars to the `common.go` file. IMO the purpose of the file is to host constants and functions to be shared between different tests.